marimbabot_vision/scripts/generate_data.py

```diff
@@ -18,9 +18,9 @@
 NUM_WORKER = 24
 OUTPUT_DIR = "data"
 MIN_DURATION = 8 # 1/8th note
-INCLUDE_DYNAMICS = False
+INCLUDE_DYNAMICS = True
 INCLUDE_SLURS = False
-INCLUDE_ARTICULATIONS = False
+INCLUDE_ARTICULATIONS = True
```

Comment:
Why did you change the default parameters?

Reply:
Because we agreed on dynamics earlier, but they were off, so we need to regenerate all the data -.- Same for the articulations. They were reduced to only marcato, but also deactivated.

Reply:
Request from Flo. The default should be the state we want to use in the end. So with dynamics and articulations (marcato)